central/cluster/datastore/datastore_impl_test.go

```diff
@@ -127,8 +129,9 @@ func (suite *ClusterDataStoreTestSuite) SetupTest() {
 	suite.clusters.EXPECT().Walk(gomock.Any(), gomock.Any()).Return(nil)
 	suite.healthStatuses.EXPECT().Walk(gomock.Any(), gomock.Any()).Return(nil)
 	suite.indexer.EXPECT().AddClusters(nil).Return(nil)
-
+	suite.imageIntegrationDataStore = imageIntegrationDatastoreMocks.NewMockDataStore(suite.mockCtrl)
```

Comment:
nit: re-add new line below this. Also, this should be with the other related initializers above